i18n/en/docusaurus-plugin-content-docs/current/features/sessions.mdx

```diff
@@ -6,94 +6,89 @@ import Screenshot from "../../../../../src/components/Screenshot";
+## Current charging session
 
-## Aktueller Ladevorgang
-
-In der Hauptansicht siehst du Informationen zum aktuellen Ladevorgang.
+In the main view you can see information about the current charging session.
 
 <Screenshot
   name="features/screenshots/sessions-main-ui"
-  caption="Messwerte am Ladevorgang"
+  caption="Measurements during charging"
 />
 
-1.  **Leistung:** Zeigt die aktuelle Ladeleistung in kW.
-    Der grüne Blitz symbolisiert, dass eine Ladung aktiv ist.
-    Unterhalb der Leistung wird die Anzahl der aktiven Phasen und deren Auslastung als kleine grüne Balken dargestellt.
+1.  **Power:** Shows the current charging power in kW.
+The green lightning symbolizes that a charge is active.
+Below the power, the number of active phases and their utilization are shown as small green bars.
 
-2.  **Geladen:** Zeigt die bereits geladene Energiemenge an.
-    Hat deine Wallbox einen Zähler verbaut, werden die Werte von dort übernommen.
-    Für Wallboxen ohne Zähler errechnet evcc die Energiemenge anhand der Ladeleistung.
+2.  **Charged:** Shows the amount of energy already charged.
+If your wallbox has a meter installed, the values ​​are taken from there.
+For wallboxes without a meter, evcc calculates the amount of energy based on the charging power.
 
-3.  **Weitere Werte:** Abhängig von deiner Installation sind noch weitere Messwerte verfügbar.
-    Diese kannst du mit Klick auf den Namen auswählen.
-    Alternativ wechselt ein Klick auf den Zahlenwert zur nächsten Anzeige.
+3.  **Other values:** Depending on your installation, other measured values ​​are available.
+You can select these by clicking on the name.
+Alternatively, clicking on the numerical value switches to the next display.
 
-        * **Restzeit:** Zeigt die verbleibende Zeit bis zum Erreichen des Ladeziels an.
-        Sofern dein Fahrzeug das unterstützt, wird die Restzeit aus den Fahrzeugdaten ermittelt.
-        Alternativ wird die Restzeit anhand der aktuellen Ladeleistung und des Ladeziels berechnet.
+        * **Remaining time:** Shows the time remaining until the charging target is reached.
+        If your vehicle supports this, the remaining time is determined from the vehicle data.
+        Alternatively, the remaining time is calculated based on the current charging power and the charging target.
 
-        * **Ladedauer:** Zeigt die effektive Dauer des Ladevorgangs an.
-        Wird der Ladevorgang pausiert (bspw. aufgrund von PV-Überschuss), wird die Zeit angehalten.
+        * **Charging duration:** Shows the effective duration of the charging session.
+        If the charging session is paused (e.g. due to PV surplus), the time is stopped.
 
-        * **Sonne:** Der Prozentwert zeigt, welcher Anteil der geladenen Energie aus der PV-Anlage stammt.
-        Energie aus dem Hausspeicher wird als Sonnenenergie gewertet.
+        * **Sun:** The percentage value shows which proportion of the charged energy comes from the PV system.
```

Comment:
![Bildschirmfoto 2024-09-15 um 08 29 03](https://github.com/user-attachments/assets/c939ea83-abcd-4047-88f7-867a12742013)

Das heißt in diesem Kontext "solar". Du kannst viele der Wordings in der Demo-Instanz https://demo.evcc.io/ sehen wenn du auf Englisch stellst.